payment/tests/test_payment.py

```diff
@@ -244,6 +262,71 @@ def test_confirmation_email_bill(self, get_order_mock):
                          soup.find('span', class_='user-full-name').text.strip())
 
 
+@override_settings(FUN_ECOMMERCE_DEBUG_NO_NOTIFICATION=False)
+@skipUnlessLms
+class RegisterUserTest(ModuleStoreTestCase):
+
+    MODULESTORE = TEST_DATA_XML_MODULESTORE
+
+    def setUp(self):
+        super(RegisterUserTest, self).setUp()
+        self.course = Course.objects.create(
+            key="edX/toy/2012_Fall",
+            title=u"course title",
+        )  # Fun Course Model
+        self.course_key = CourseKey.from_string(self.course.key)
+        self.user = UserFactory(
+            username="user",
+            first_name="first_name",
+            last_name="last_name",
+            email="richard@example.com",
+            is_active=True,
+        )
+        cohort = CourseUserGroup.objects.create(
+            name="Certificat",
+            course_id=self.course_key,
+            group_type=CourseUserGroup.COHORT,
+        )
+
+    @patch("payment.utils.add_user_to_cohort")
+    def test_register_user_no_verified_cohort(self, add_user_to_cohort_mock):
+        register_user_verified_cohort(
```

Comment:
Can you add docstring on all your tests to describe what it is testing? e.g. here probably something like:
```
"""We should not attempt to add user to cohort if the VERIFIED_COHORTS setting is not set."""
*```